Gems/ImGui/Code/Source/LYCommonMenu/ImGuiLYCommonMenu.cpp

```diff
@@ -113,6 +114,50 @@ namespace ImGui
             viewportBorderPaddingOpt, &AzFramework::ViewportBorderRequestBus::Events::GetViewportBorderPadding);
 
         AzFramework::ViewportBorderPadding viewportBorderPadding = viewportBorderPaddingOpt.value_or(AzFramework::ViewportBorderPadding{});
+
+        auto ctx = ImGui::GetCurrentContext();
+
+        // determine if the window is the drop down and if it is active
+        const auto windowIt = AZStd::find_if(
+            ctx->Windows.begin(),
+            ctx->Windows.end(),
+            [](const ImGuiWindow* window)
+            {
+                // there is a drop down if the BeginOrder is 2 (1 below main menu),
+                // drop down items are called "##Menu_00"
+                return window->BeginOrderWithinContext == 2 && strcmp(window->Name, "##Menu_00") == 0 && window->WasActive;
```

Comment:
This looks good but may potentially be fragile, I think fine for now but something to keep an eye on, good find getting something that works reliably 👍